packages/editor/src/utils/badge-properties.ts

```diff
@@ -18,10 +18,10 @@ const badgePropertyCMS = {
   badgeTextGen: (text: string) => text.replaceAll(/#{\s*ivy.cms.co\(['"]|\/*[^\\]+\/|['"]\)\s*}/g, '')
 };
 
-const badgePropertyCondition = {
-  regex: /#{\s*[^\s}]+(?:\s+[^\s}]+)+\s*}/,
+const badgePropertyExpression = {
+  regex: /#{[^}]*}/,
   icon: IvyIcons.StartProgram,
-  badgeTextGen: (text: string) => text.replaceAll(/#{\s*|}/g, '')
+  badgeTextGen: (text: string) => text.replaceAll(/#{\s*|\s*}/g, '')
 };
 
-export const badgeProps = [badgePropertyData, badgePropertyLogic, badgePropertyCMS, badgePropertyCondition];
+export const badgeProps = [badgePropertyExpression, badgePropertyData, badgePropertyLogic, badgePropertyCMS];
```

Comment:
how is the order of this regex expressions?
if I understand this correctly the badgePropertyExpression is the fallback, so the first one is the fallback?
And by the way do you have tests for this regular expressions? or better for the combination of all of them?

Reply:
I've now added a test for all the badges :smile:

As it is implemented now, the last prop in the array has the highest priority, that is why the "fallback" (Expression) is in first place.

Reply:
Strange order for me...
I would expect the first the highest etc.